include/odp/api/spec/ipsec.h

```diff
@@ -1224,16 +1224,22 @@ typedef struct odp_ipsec_status_t {
  *
  * The operation does packet transformation according to IPSEC standards (see
  * e.g. RFC 4302 and 4303). Resulting packets are well formed, reconstructed
- * original IP packets, with IPSEC headers removed and valid header field values
- * restored. The amount and content of packet data before the IP header is
- * undefined.
+ * original IP (or TFC dummy) packets, with IPSEC headers removed and valid
+ * header field values restored. The amount and content of packet data before
+ * the IP header is undefined. TFC padding may follow the IP packet payload,
+ * in which case packet length is larger than protocol headers indicate.
+ * TFC dummy packets in tunnel mode have both IPv4 and IPv6 flags cleared and
+ * may contain some (dummy) data.
```

Comment:
Also it should be documented that in INLINE mode hardware might drop dummy packets on its own, without reporting them to an application.